zerver/lib/actions.py

```diff
@@ -2023,13 +2023,23 @@ def do_add_default_stream(realm, stream_name):
     stream, _ = create_stream_if_needed(realm, stream_name)
     if not DefaultStream.objects.filter(realm=realm, stream=stream).exists():
         DefaultStream.objects.create(realm=realm, stream=stream)
+        event = dict(
+            type='update_default_streams',
+            default_streams=do_get_default_streams_for_realm(realm)
```

Comment:
There should be whitespace on either side of `=` in the Zulip Python style, can you fix this here and elsewhere in this PR?